src/stillir.app.src

```diff
@@ -2,8 +2,11 @@
 {application, stillir,
  [
   {description, "Read Unix env vars into application config"},
-  {vsn, "1.0.0"},
+  {vsn, git},
   {registered, []},
   {applications, [kernel,
-                  stdlib]}
+                  stdlib]},
+  {maintainers, ["Fred Hebert"]},
```

Comment:
Fred Hebert isn't with Heroku any more. Can you change this to include Jamu Kakar (hex user: jkakar) and Tilman Holschuh (hex user: ypaq).